### PR TITLE
[FIX] web_editor: Fix style formating traceback

### PR DESCRIPTION
**Pull request by sebgeelen**
In some case, when removing format on element with multiple textNode,
the code generate a traceback.

The loops in `toggleFormat` provide a limit element `closestDecorated` to
the `splitAroundUntil` util? However this limit element can be split and
dumped by the format removal. In this case the reference was never encountered
in the DOM by the `splitAroundUntil` code and will bubble all the way to the
`<html>` TAG an try to split it, which generate an error.

We Fix it by getting the `closestDecorated` in every loop.

task-2850686

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/static/lib/odoo-editor/src/commands/commands.js [hunk 1]
```diff
@@ -403,12 +403,12 @@ export function toggleFormat(editor, format) {
     const {anchorNode, anchorOffset, focusNode, focusOffset} = editor.document.getSelection();
     const style = styles[format];
     const selectedTextNodes = getSelectedNodes(editor.editable)
-        .filter(n => n.nodeType === Node.TEXT_NODE && n.nodeValue.trim().length);
-    const isAlreadyFormatted = style.is(editor.editable);
+        .filter(n => n.nodeType === Node.TEXT_NODE && n.nodeValue.length);
+    const isAlreadyFormatted = style.isInSelection(editor.editable);
```
Comment (dmo-odoo) [inline addons/web_editor/static/lib/odoo-editor/src/commands/commands.js:408]: I don't understand the reason of this renaming.

File: addons/web_editor/static/lib/odoo-editor/src/commands/commands.js [hunk 2]
```diff
@@ -363,27 +363,27 @@ export function applyInlineStyle(editor, applyStyle, style, shouldApply=true) {
 }
```
Comment (dmo-odoo) [inline addons/web_editor/static/lib/odoo-editor/src/commands/commands.js:363]: About the commit message: I'm not sure the `?` at the end of the first sentence of the second paragraph is intentional ? If it is then I don't undersand the sentence... ^^' Also, I'd remove the capital letter of "Fix" in the last paragraph.

File: addons/web_editor/static/lib/odoo-editor/src/commands/commands.js (hunk 1 above)
Comment (sebgeelen) [reply addons/web_editor/static/lib/odoo-editor/src/commands/commands.js:408]: I didn't understand the is when I read the code. So I rename it to be more clear.

Simple as that.

File: addons/web_editor/static/lib/odoo-editor/src/commands/commands.js [hunk 3]
```diff
@@ -284,11 +284,11 @@ export function applyInlineStyle(editor, applyStyle, style, shouldApply=true) {
     const [
         normalizedStartContainer,
         normalizedStartOffset
-    ] = getNormalizedCursorPosition(startContainer, startOffset)
+    ] = getNormalizedCursorPosition(startContainer, startOffset);
```
Comment (dmo-odoo) [inline addons/web_editor/static/lib/odoo-editor/src/commands/commands.js:287]: Commit message: `[FIX] web_editor: fix some linting errors`

**Comment by dmo-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**